ui/tests/unit/models/node-test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Model | NodePool', function (hooks) {
+  setupTest(hooks);
+
+  test('a node pool can be associated with multiples nodes', function (assert) {
```

Comment:
```suggestion
  test('a node pool can have multiples nodes', function (assert) {
```

Minor nit-picking, but I think this describes the expected relationship better.